modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -257,19 +265,18 @@ def location(type, appointment_hash)
 
         def location_phone(appointment_hash)
           phone = appointment_hash.dig(:location, :phone, :main)
-          return nil unless phone
 
           # captures area code (\d{3}) number (\d{3}-\d{4})
           # and optional extension (until the end of the string) (?:\sx(\d*))?$
-          phone_captures = phone.match(/^(\d{3})-(\d{3}-\d{4})(?:\sx(\d*))?$/)
+          phone_captures = phone&.match(/^(\d{3})-(\d{3}-\d{4})(?:\sx(\d*))?$/)
 
           if phone_captures.nil?
             Rails.logger.warn(
               'mobile appointments failed to parse VAOS V2 facility phone number',
               facility_id: appointment_hash.dig(:location, :id),
               facility_phone: phone
             )
-            return nil
+            return { area_code: nil, number: nil, extension: nil }
```

Comment:
I probably should've caught this in review, given that I had to set this behavior elsewhere.